### ISSUE TITLE
Unable to Fetch Series Data (if show is in fact empty), data.season=null

### ISSUE THREAD

**Issue by Baine** [HristoKolev/TvDbSharper#16]
we've got TvDbSharper as helper module to retrieve information about (Anime-)Series from TVDB.

slim version of the code:
```csharp
    public static class TvDBApiHelper
    {
        private static readonly ITvDbClient client;
        static TvDBApiHelper()
        {
            client=new TvDbClient();
            client.BaseUrl = "https://api-beta.thetvdb.com";
        }

        await client.Authentication.AuthenticateAsync(Constants.TvDB.apiKey);
        var response = await client.Series.GetAsync(seriesID);
   }
```

while testing i selected a show without any data by mistake
(https://www.thetvdb.com/series/digimon-adventure)
, causing the following error to be thrown:
```
Error|ExecutionContext.Run => ExecutionContext.RunInternal => <GetSeriesInfoOnlineAsync>d__7.MoveNext Error in TvDBApiHelper.GetSeriesInfoOnline: Newtonsoft.Json.JsonSerializationException: Error converting value {null} to type 'System.Int32'. Path 'data.season', line 1, position 77. ---> System.InvalidCastException: Null object cannot be converted to a value type.
```
maybe there's a better way to handle this? (like nullable int or something similar)

**Comment by HristoKolev** [comment]
Yes, this is a bug. Will be releasing bug fix soon.

**Comment by HristoKolev** [comment]
Fixed in 3.0.1. Please, create a new issue if you find something else. :)

**Comment by Baine** [comment]
thank you :) will do if i should find anything

Am Di., 14. Jan. 2020 um 18:35 Uhr schrieb Hristo Kolev <
notifications@github.com>:
